xija/component/heat.py

```diff
@@ -128,6 +128,8 @@ def __init__(self, model, node, pitch_comp, eclipse_comp=None,
             dPs = np.zeros_like(self.P_pitches)
         self.dPs = np.array(dPs, dtype=np.float)
 
+        if epoch is None:
```

Comment:
This needs to be verified with all flight models since it could cause breakage.  The PSMC model does not currently have `epoch` set in the spec file, but it also does not use `SolarHeat`.